stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -428,11 +455,12 @@ function externEncryptAesGcm(byte[] input, byte[] key, byte[] iv, handle padding
 } external;
 
 # Returns RSA decrypted value for the given RSA encrypted data.
+# ```ballerina byte[]|crypto:Error plainText = crypto:decryptRsaEcb(cipherText, privateKey); ```
 #
 # + input - The content to be decrypted
 # + key - Private or public key used for encryption
 # + padding - The padding
-# + return - Decrypted data or `Error` if key is invalid
+# + return - Decrypted data or `crypto:Error` if key is invalid
```

Comment:
```suggestion
# + return - Decrypted data or `crypto:Error` if key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.